restaurant/settings.py

```diff
@@ -138,4 +138,11 @@
 # https://docs.djangoproject.com/en/1.11/howto/static-files/
 
 STATIC_URL = '/static/'
-STATIC_ROOT = os.path.join(BASE_DIR, 'static')
\ No newline at end of file
+STATIC_ROOT = os.path.join(BASE_DIR, 'static')
+
+#User's model
+AUTH_USER_MODEL = 'administrator.UserProfile'
+
+#Files downloaded by the users of the site
+MEDIA_URL = '/media/'
+MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
```

Comment:
Maybe it's better to add a new line at the end of the file? This will prevent the line from being displayed as modified in future.